generators/php/math.js

```diff
@@ -172,34 +214,25 @@ Blockly.PHP['math_number_property'] = function(block) {
          '  }',
          '  return true;',
          '}']);
-    code = functionName + '(' + number_to_check + ')';
-    return [code, Blockly.PHP.ORDER_FUNCTION_CALL];
-  }
-  switch (dropdown_property) {
-    case 'EVEN':
-      code = number_to_check + ' % 2 == 0';
-      break;
-    case 'ODD':
-      code = number_to_check + ' % 2 == 1';
-      break;
-    case 'WHOLE':
-      code = 'is_int(' + number_to_check + ')';
-      break;
-    case 'POSITIVE':
-      code = number_to_check + ' > 0';
-      break;
-    case 'NEGATIVE':
-      code = number_to_check + ' < 0';
-      break;
-    case 'DIVISIBLE_BY': {
-      const divisor = Blockly.PHP.valueToCode(block, 'DIVISOR',
-          Blockly.PHP.ORDER_MODULUS) || '0';
-      code = number_to_check + ' % ' + divisor + ' == 0';
-      break;
-    }
-  }
-  return [code, Blockly.PHP.ORDER_EQUALITY];
-};
+         code = functionName + '(' + numberToCheck + ')';
```

Comment:
This should only be indented by four spaces to line up with `const functionName` above.